README.md

```diff
@@ -33,6 +33,14 @@ philosophy](https://abseil.io/about/philosophy).
 `prototype-kicad-library-ci` uses the [GitHub
 flow](https://guides.github.com/introduction/flow/) workflow.
 
+## Git Config Install Instructions
```

Comment:
Please change the subsection title to `Git Config` (as was requested in #13) for consistency with the `Git Hooks` subsection title.
```suggestion
## Git Config
```